### PR TITLE
NMS-14674 Set Up Enlinkd schedule time interval based on protocols

### PR DESCRIPTION
**Pull request by rssntn67**
Schedule is performed using protocol rescan interval

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-14764

### CODE REVIEW

File: docs/modules/operation/pages/deep-dive/topology/enlinkd/introduction.adoc [hunk 1]
```diff
@@ -53,8 +53,10 @@ It also includes xref:development:rest/user-defined-links.adoc[User-Defined] top
 == Enlinkd daemon
 
 Essentially, each Enlinkd-Collector asks each device the following question: "What is the network topology from your point of view?"
-The Enlinkd-Discovery process attempts to discover bridge domain links with the data coming from all collected Bridge Forwarding Tables.
-The Enlinkd-Updaters correlate all collected topology data to generate a global topology layout of your network.
+CDP, LLDP, OSPF and ISIS collectors provide the required information about links.
+Bridge Collectors are able to collect Bridge Forwarding Table and IpNetToMedia Table on devices and to provide effective links a further step is required.
+The Enlinkd-Bbridge-Discovery process attempts to discover bridge domain links with the data coming from all collected Bridge Forwarding Tables.
```
Comment (christianpape) [inline docs/modules/operation/pages/deep-dive/topology/enlinkd/introduction.adoc:58]:
```suggestion
The Enlinkd-Bridge-Discovery process attempts to discover bridge domain links with the data coming from all collected Bridge Forwarding Tables.
```

**Review by christianpape — CHANGES_REQUESTED** [review]
Only a typo.

**Review by christianpape — APPROVED** [review]
LGTM.

**Review by aramosvizcarra — APPROVED** [review]
LGTM!

**Comment by Bonrob2** [comment]
No major changes from me, but I would have liked to review the docs before the PR was merged.

**Comment by rssntn67** [comment]
> No major changes from me, but I would have liked to review the docs before the PR was merged.

I apologize. I was in a hurry, because I'm committing long term work. I splitted my big one (93 file changed) in small PR so that reviewers will have a better look at the changes...
In any case I'm going to create a new Issue for update enlinkd configuration.
In fact I'm changing the configuration file to use priority execution.

**Comment by Bonrob2** [comment]
> I apologize. I was in a hurry, because I'm committing long term work. I splitted my big one (93 file changed) in small PR so that reviewers will have a better look at the changes... In any case I'm going to create a new Issue for update enlinkd configuration. In fact I'm changing the configuration file to use priority execution.

No problem. I totally understand. And it read fine anyway. I might have made a few tweaks, but nothing major. It's great that you took the time to write and update the docs!

**Pull request merged**